src/main/java/frc/robot/subsystems/superstructure/shooter/ShooterIOReal.java

```diff
@@ -181,17 +181,17 @@ public void updateInputs(final ShooterIO.ShooterIOInputs inputs) {
 
         inputs.ampPositionRots = _ampPosition.getValue();
         inputs.ampVelocityRotsPerSec = _ampVelocity.getValue();
-        inputs.ampVoltageVolts = _ampVoltage.getValue();
+        inputs.ampVoltage = _ampVoltage.getValue();
```

Comment:
There's no reason to remove the unit, right? Voltage doesn't imply volts, as it can be MV, mV, KV, etc...

Reply:
Also, this is in shooter? This PR should be related to the pivot only.

Reply:
Its always volts in frc bruh...

Reply:
Also not sure what commit that's from cause I didn't change that when I added the cancoder

Reply:
Should be rolled back then if its related to the shooter.

Reply:
Still needs to be rolled back.